proto/rill/admin/v1/api.proto

```diff
@@ -58,6 +58,11 @@ service AdminService {
     option (google.api.http) = {get: "/v1/organizations/{org_name}/billing/subscriptions"};
   }
 
+  // GetOrganizationBillingSessionURL returns the URL for the billing session to collect payment method
+  rpc GetOrganizationBillingSessionURL(GetOrganizationBillingSessionURLRequest) returns (GetOrganizationBillingSessionURLResponse) {
+    option (google.api.http) = {get: "/v1/organizations/{org_name}/billing/payments/session-url"};
+  }
```

Comment:
Can we merge this with `GetOrganizationBillingSubscriptionResponse`, i.e. just return the payment portal URL alongside the billing portal URL?

Reply:
The reason this is a separate API because billing customer portal URL is fixed (for Orb atleast) and does not change where as this  payment session url expires and needs to be created on-demand using stripe API. I wanted to unnecessarily create this every time we fetch subscription details which can be done multiple times for various use cases.

Reply:
Okay, can we then rename to clarify the difference – e.g. `GetPaymentsPortalURL` ("session URL" does not seem very descriptive).

And on reflection, I don't think we need to prefix these with calls with "organization". Should we also change:
- `GetOrganizationBillingSubscription` -> `GetBillingSubscription`
- `UpdateOrganizationBillingSubscription` -> `UpdateBillingSubscription`